### PR TITLE
Add benchmark for packed texture on GPU

### PR DESCRIPTION
**Pull request by Lewuathe**
Add missing benchmark of packed texture on GPU. It can show the speed that is significantly faster than unpacked texture.

screen shot 2017-09-10 at 23 09 02

---
This change is [Reviewable](https://reviewable.io/reviews/pair-code/deeplearnjs/115)

### CODE REVIEW

**Comment by nsthorat** [comment]
So this originally was in the benchmarks (which is why the actual file exists), but I removed it since we're going to implement this optimization in a different way. I don't want to get people's hopes up with the code as is.

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/115#-:-KtlcJOGD1J_kbV2DzMT:bc9hke0)*

**Comment by Lewuathe** [comment]
I see. `MAX_POOL_POSNS_BENCHMARK_TEST` in max pool benchmark is not used either. Is this from same reason?

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pair-code/deeplearnjs/115#-:-Ktow51e8GAdFJdkFvRi:ba7fajd)*

**Comment by nsthorat** [comment]
Similar reason, this is just not useful to benchmark (it's only one part of the backprop).

**Comment by nsthorat** [comment]
There are lots of other GPU methods we just haven't written benchmarks for which would be interesting to benchmark (unary ops, binary ops, resizing, batchnorm etc).
